x/bank/simulation/msgs.go

```diff
@@ -64,7 +64,7 @@ func createSingleInputSendMsg(r *rand.Rand, ctx sdk.Context, accs []simulation.A
 		toAcc = simulation.RandomAcc(r, accs)
 	}
 	toAddr := toAcc.Address
-	initFromCoins := mapper.GetAccount(ctx, fromAcc.Address).GetCoins()
+	initFromCoins := mapper.GetAccount(ctx, fromAcc.Address).SpendableCoins(ctx.BlockHeader().Time)
```

Comment:
Note we need to make sure we only send "sendable" amount of coins.

Reply:
Ah - but sometimes (randomly) I think we should try to spend more than we can!

Reply:
If we try to send any more than "sendable", then `subtractCoins` will panic?

Reply:
If the transaction tries to spend more, it should fail - right?

Reply:
yes, it'll fail and panic at `subtractCoins`.